notifications/src/main/java/dk/shape/games/notifications/presentation/viewmodels/notifications/SubjectNotificationSwitcherViewModel.kt

```diff
@@ -26,12 +27,23 @@ internal class SubjectNotificationSwitcherViewModel(
             )
     )
 
-    val item: ObservableField<Any> = ObservableField<Any>().onChange {
-        onItemChanged()
+    val item: ObservableField<Any> = ObservableField<Any>(loadingViewModel).onChange {
+        if (it is SubjectNotificationViewModel.Content) {
```

Comment:
should this if be combined?
```suggestion
        if (it is SubjectNotificationViewModel.Content && lastViewModel !is SubjectNotificationViewModel.Skeleton) {
```

Reply:
I nested it to not have a long if. Only for aesthetics . But it is virtually the same

Reply:
It looks clearer to me if combined on a single if, otherwise it feels like we are missing handling other cases